releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,17 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.1.0-preview - Jun XX, 2023
+
+### New Features
+
+* Dynamic configuration support, support refreshing the data of ConfigMap or Secret by watching the sentinel key periodically.

```

Comment:
Added support for dynamic configuration so the corresponding ConfigMap and Secret will be automatically updated when data is changed in App Configuration.